cmd/oras/internal/option/target.go

```diff
@@ -148,18 +150,27 @@ func (opts *Target) NewReadonlyTarget(ctx context.Context, common Common, logger
 	switch opts.Type {
 	case TargetTypeOCILayout:
 		var err error
+
 		opts.Path, opts.Reference, err = parseOCILayoutReference(opts.RawReference)
 		if err != nil {
 			return nil, err
 		}
 		info, err := os.Stat(opts.Path)
 		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				err = fmt.Errorf("invalid path in %q: %w", opts.RawReference, err)
```

Comment:
How about something like
```
path "foo" in "foo:bar" cannot be found
```

Reply:
I think `cannot find path "foo" in "foo:bar"` reads a bit better

Reply:
Trying to mention `path "foo" in "foo:bar"` here because such statement is ambiguous, user might think ORAS CLI cannot find valid path (foo) in the provided argument (foo:bar). How about 
> Error: invalid arguent "foo:bar": failed to find path "foo": stat foo: no such file or directory

Or swallow the stat error, just
> Error: invalid arguent "foo:bar": failed to find path "foo": no such file or directory